src/browser/bounds_changed_state_tracker.ts

```diff
@@ -0,0 +1,592 @@
+/*
+    src/browser/bounds_changed_state_tracker.js
+ */
+
+const WindowTransaction = require('electron').windowTransaction;
+
+import * as _ from 'underscore';
+import * as animations from './animations.js';
+import * as coreState from './core_state.js';
+import * as os from 'os';
+import * as Deferred from './deferred';
+import WindowGroups from './window_groups';
+import WindowGroupTransactionTracker, { ITransaction } from './window_group_transaction_tracker';
+import { toSafeInt } from '../common/safe_int';
+import ofEvents from './of_events';
+import route from '../common/route';
+import { clipBounds, windowSetBoundsToVisible } from './utils';
+import { OpenFinWindow, BrowserWindow } from '../shapes';
+import { Rectangle, windowTransaction } from 'electron';
```

Comment:
What's the difference between this `windowTransaction` and the `WindowTransaction` on line 5

Reply:
One is the typescript stub definition, the other is the js that actually runs. Having them not be the same isn't ideal, but I didn't want to change what other files already use.